extensions/ql-vscode/src/data-extensions-editor/data-extensions-editor-view.ts

```diff
@@ -479,4 +483,19 @@ export class DataExtensionsEditorView extends AbstractWebview<
       message: "",
     });
   }
+
+  private async callApi(request: ModelRequest): Promise<ModelResponse | null> {
+    try {
+      return await autoModel(this.app.credentials, request);
+    } catch (e) {
+      if (e instanceof RequestError && e.status === 429) {
+        void showAndLogExceptionWithTelemetry(
+          redactableError`Rate limit hit, please try again soon.`,
```

Comment:
We should probably also report the full error:

```suggestion
          redactableError(e)`Rate limit hit, please try again soon.`,
```